public/API/API_GetGameExtended.php

```diff
@@ -50,28 +50,100 @@
  *    string    Expiration                date the claim will expire
  */
 
+use App\Models\Achievement;
+use App\Models\AchievementSetClaim;
+use App\Models\Game;
 use App\Platform\Enums\AchievementFlag;
+use Carbon\Carbon;
 
-$gameID = (int) request()->query('i');
+$gameId = (int) request()->query('i');
 $flag = (int) request()->query('f', (string) AchievementFlag::OfficialCore);
-getGameMetadata($gameID, null, $achData, $gameData, flag: $flag, metrics: true);
 
-if ($gameData === null) {
+$game = Game::with('system')->find($gameId);
+
+if (!$game) {
     return response()->json();
 }
 
-if (empty($achData)) {
-    $gameData['Achievements'] = new ArrayObject(); // issue #484 - force serialization to {}
+$gameSetClaims = AchievementSetClaim::where('GameID', $gameId)->get();
+$gameAchievements = Achievement::where('GameID', $gameId)->where('Flags', $flag)->findMany($game->achievements);
+
+$gameData = [
+    'ID' => $game->ID,
+    'Title' => $game->Title,
+    'ConsoleID' => $game->ConsoleID,
+    'ForumTopicID' => $game->ForumTopicID,
+    'Flags' => null, // Always '0', this is different in the extended endpoint test for some reason
+    'ImageIcon' => $game->ImageIcon,
+    'ImageTitle' => $game->ImageTitle,
+    'ImageIngame' => $game->ImageIngame,
+    'ImageBoxArt' => $game->ImageBoxArt,
+    'Publisher' => $game->Publisher,
+    'Developer' => $game->Developer,
+    'Genre' => $game->Genre,
+    'Released' => $game->Released,
+    'IsFinal' => $game->IsFinal,
+    'RichPresencePatch' => md5($game->RichPresencePatch),
+    'GuideURL' => $game->GuideURL,
+    'Updated' => $game->Updated,
```

Comment:
If I'm not mistaken, this is a Carbon date too. While dumping it natively works, we may want to manually use `->format()` just to be safe.

Reply:
Close, the format being output by the latest code revision is:

"2024-02-24 17:33:55"

To match prod, we need:

"2024-04-02T19:52:54.000000Z"

Might just be a matter of `->format('Y-m-d\TH:i:s.u\Z')`.

Reply:
Did some googling and found that `toIso8601ZuluString()` is also a potential solution though it doesn't add the extra 0's between the Seconds and the Z so I'll go with the one that you suggested.